### DISCUSSION TITLE
How to use MongoDB directly with Nightscout on Google Cloud (without migrating/copying)

### DISCUSSION THREAD

**Discussion by szantos** [NightscoutFoundation/xDrip#2781] [Q&A] [answered]
I'd like to test Nightscout (NS) on Google Cloud (GC) with an easy option to switch to another provider. That is the reason I won't transfer the DB to GC.

I have set up NS on GC according https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud. Thanks for the guide! 

I've created the same MONGODB_URI key with the same value like in Heroku and Azure, but it did not work.

Moreover a weird situation happened. The name of the key became (yes, this long):
MONGODB_URI=mongodb+srv://xxxUSER:xxx@xxx.mongodb.net/xxx?retryWrites=true&w=

And the value became only:
majority

I've even tried to create the variable in the text editor with the same syntax like any other variable. Checking the value in the web editor showed the same result only 'majority'.

What is the problem here? Can you please help me, please?

**Comment by Navid200** [comment]
@tzachi-dar Any thoughts?

**Comment by Navid200** [comment, marked as answer]
The intent of our project has been to have the database on the virtual machine.
Therefore, our project does not support having the database on a remote server.

**Reply by szantos** [reply]
Thank you for the clarification!
